{"name": "test_add[bigint]", "status": "broken", "statusDetails": {"message": "NameError: name 'add_function' is not defined", "trace": "a = 100000, b = 100000, expected = 200000\n\n    @pytest.mark.parametrize(\"a,b,expected\",\n                             [(3,5,8),\n                              (-1,-2,-3),\n                              (100000,100000,200000)\n                              ],ids = [\"int\",\"minus\",\"bigint\"]\n                             )\n    def test_add(a,b,expected):\n>       assert  add_function(a,b) == expected\nE       NameError: name 'add_function' is not defined\n\ntest_parametrize.py:23: NameError"}, "parameters": [{"name": "a", "value": "100000"}, {"name": "b", "value": "100000"}, {"name": "expected", "value": "200000"}], "start": 1618644300968, "stop": 1618644300969, "uuid": "7765be23-cbd4-41c3-9251-9e2639fa6841", "historyId": "3efe42854320ac2f127f7933f76211c2", "testCaseId": "6f41edd1e423772ab3257ab38ec97795", "fullName": "test_parametrize#test_add", "labels": [{"name": "suite", "value": "test_parametrize"}, {"name": "host", "value": "BY-20190922YINQ"}, {"name": "thread", "value": "14192-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_parametrize"}]}